src/WordPress/CLI/CLI.php

```diff
@@ -31,4 +33,22 @@ public function __invoke()
 
         WP_CLI::log('Iris complete.');
     }
+
+    public static function convertById($id)
+    {
+        $image_id = $id[0];
```

Comment:
is it worth adding an empty check on `$id[0]` do you think and doing a `WP_CLI::error` if there isn't one?